tests/vmi_multus_test.go

```diff
@@ -1029,6 +1042,19 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 	return setInterfaceUp(vmi, interfaceName)
 }
 
+func configureSRIOVInterfaceStaticIPByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, interfaceMac, interfaceAddress string) error {
```

Comment:
Please drop the `SRIOV`, the helper works on any interface.
In general, try to avoid naming functions by the context of the caller, the should express what they do regardless of who is calling them.

Reply:
Done.